tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -5032,10 +5036,39 @@ <h2>Show similarity to selected datapoint</h2>
        */
       addChart: function(chartType, featureName, data){
         let chart;
+        let dataToChart = data;
+
+        // If multiclass and there are more inferred classes than the set
+        // number of classes to display, then filter the classes to display
+        // in the chart down to the top scoring classes.
+        if (this.isMultiClass_(this.modelType, this.multiClass)) {
+          dataToChart = [];
+          for (let modelIdx = 0; modelIdx < data.length; modelIdx++) {
+            let modelDataToChart = {};
+            if (Object.keys(data[modelIdx]).length >
+                parseInt(this.maxInferenceEntriesPerRun)) {
+              const exampleId = this.selected &&
+                this.selected.length > 0 ? this.selected[0] : 0;
+              // Get the class labels of the top scoring classes from the most
+              // recent inference.
+              const labels = this.examplesAndInferences[exampleId].inferences[
```

Comment:
Woah, this is a bit hard to read and parse. How about something like
```ts
const currentExample = this.examplesAndInferences[exampleId];
const lastInference = currentExample.inferences[currentExample.inferences.length - 1];
// or 
// const lastInference = currentExample.inferences.slice(-1)[0];
const labels = lastInference[modelIdx].slice(0, this.maxInferenceEntriesPerRun);
```

Reply:
done